docs/_langs.md

```diff
@@ -2,3 +2,4 @@
   - [:uk: English](/)
   - [:cn: 中文](/zh-cn/)
   - [:fr: Français](/fr-fr/)
+  - [🇹🇭 ไทย](/th-th/)
```

Comment:
The GitHub emoji code for the Thai flag is `:thailand:` however this doesn't seem to work in Docsify.

[There is an open PR](https://github.com/docsifyjs/docsify/pull/930) for extending the emoji support, but the URL that would be generated for the Thai flag 404s. Maybe we should just use the Unicode character directly for all languages as has been done here.